ecosystem/solutions-intro.md

```diff
@@ -0,0 +1,4 @@
+Solutions integrate Trino to create a full-featured product for specific use
+cases. They can simplify Trino management, provide additional features and
+extensions such as security and access control enhancement, or adapt Trino to
```

Comment:
```suggestion
extensions, such as security and access control enhancement, or adapt Trino to
```

We're duplicating this text in solution.md and solutions-intro.md. Is that correct?

Reply:
Opps .. fixed